[2024-12-05T20:26:51.314+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-12-05T20:26:51.328+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ELT_dbt.dbt_run manual__2024-12-05T20:26:50.679649+00:00 [queued]>
[2024-12-05T20:26:51.332+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ELT_dbt.dbt_run manual__2024-12-05T20:26:50.679649+00:00 [queued]>
[2024-12-05T20:26:51.332+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-12-05T20:26:51.337+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): dbt_run> on 2024-12-05 20:26:50.679649+00:00
[2024-12-05T20:26:51.342+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'ELT_dbt', 'dbt_run', 'manual__2024-12-05T20:26:50.679649+00:00', '--job-id', '507', '--raw', '--subdir', 'DAGS_FOLDER/ELT_metrics.py', '--cfg-path', '/tmp/tmp7p7rpo_s']
[2024-12-05T20:26:51.343+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=548) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-05T20:26:51.343+0000] {standard_task_runner.py:91} INFO - Job 507: Subtask dbt_run
[2024-12-05T20:26:51.343+0000] {standard_task_runner.py:63} INFO - Started process 549 to run task
[2024-12-05T20:26:51.366+0000] {task_command.py:426} INFO - Running <TaskInstance: ELT_dbt.dbt_run manual__2024-12-05T20:26:50.679649+00:00 [running]> on host bd7c71f68e16
[2024-12-05T20:26:51.408+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ELT_dbt' AIRFLOW_CTX_TASK_ID='dbt_run' AIRFLOW_CTX_EXECUTION_DATE='2024-12-05T20:26:50.679649+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-05T20:26:50.679649+00:00'
[2024-12-05T20:26:51.409+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-12-05T20:26:51.417+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-12-05T20:26:51.418+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '/home/***/.local/bin/dbt run --profiles-dir /opt/***/dbt_proj --project-dir /opt/***/dbt_proj']
[2024-12-05T20:26:51.422+0000] {subprocess.py:86} INFO - Output:
[2024-12-05T20:26:52.617+0000] {subprocess.py:93} INFO - [0m20:26:52  Running with dbt=1.8.7
[2024-12-05T20:26:53.189+0000] {subprocess.py:93} INFO - [0m20:26:53  Registered adapter: snowflake=1.8.4
[2024-12-05T20:26:53.416+0000] {subprocess.py:93} INFO - [0m20:26:53  Found 4 models, 3 snapshots, 4 data tests, 5 sources, 459 macros
[2024-12-05T20:26:53.418+0000] {subprocess.py:93} INFO - [0m20:26:53
[2024-12-05T20:26:55.130+0000] {subprocess.py:93} INFO - [0m20:26:55  Concurrency: 1 threads (target='dev')
[2024-12-05T20:26:55.130+0000] {subprocess.py:93} INFO - [0m20:26:55
[2024-12-05T20:26:55.142+0000] {subprocess.py:93} INFO - [0m20:26:55  1 of 2 START sql table model analytics.citibike_historical_summary ............. [RUN]
[2024-12-05T20:26:56.581+0000] {subprocess.py:93} INFO - [0m20:26:56  1 of 2 OK created sql table model analytics.citibike_historical_summary ........ [[32mSUCCESS 1[0m in 1.44s]
[2024-12-05T20:26:56.586+0000] {subprocess.py:93} INFO - [0m20:26:56  2 of 2 START sql table model analytics.weather_current_summary ................. [RUN]
[2024-12-05T20:26:58.066+0000] {subprocess.py:93} INFO - [0m20:26:58  2 of 2 OK created sql table model analytics.weather_current_summary ............ [[32mSUCCESS 1[0m in 1.48s]
[2024-12-05T20:26:58.069+0000] {subprocess.py:93} INFO - [0m20:26:58
[2024-12-05T20:26:58.070+0000] {subprocess.py:93} INFO - [0m20:26:58  Finished running 2 table models in 0 hours 0 minutes and 4.65 seconds (4.65s).
[2024-12-05T20:26:58.104+0000] {subprocess.py:93} INFO - [0m20:26:58
[2024-12-05T20:26:58.104+0000] {subprocess.py:93} INFO - [0m20:26:58  [32mCompleted successfully[0m
[2024-12-05T20:26:58.104+0000] {subprocess.py:93} INFO - [0m20:26:58
[2024-12-05T20:26:58.105+0000] {subprocess.py:93} INFO - [0m20:26:58  Done. PASS=2 WARN=0 ERROR=0 SKIP=0 TOTAL=2
[2024-12-05T20:26:58.677+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-12-05T20:26:58.677+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-12-05T20:26:58.691+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=ELT_dbt, task_id=dbt_run, run_id=manual__2024-12-05T20:26:50.679649+00:00, execution_date=20241205T202650, start_date=20241205T202651, end_date=20241205T202658
[2024-12-05T20:26:58.707+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-12-05T20:26:58.719+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-05T20:26:58.720+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
